package.json

```diff
@@ -11,50 +11,50 @@
     "vanilla/*"
   ],
   "scripts": {
-    "bootstrap": "lerna bootstrap",
-    "lerna": "lerna",
-    "clean": "lerna run --parallel clean",
-    "build": "lerna run --parallel build",
-    "lint": "lerna run --parallel lint:es",
-    "lint:fix": "lerna run --parallel lint:es:fix",
-    "tsc": "lerna run --parallel tsc",
-    "test": "lerna run --parallel test",
+    "clean": "npm run clean --workspaces --if-present",
+    "clean:all": "rm -rf node_modules && npm run clean:all --workspaces --if-present",
```

Comment:
Superfluous.

Reply:
Again, convenience for me.

This allows me test a clone and install that a customer might do. I was using extensively testing out this PR.